app/controllers/assessments_controller.rb

```diff
@@ -2000,78 +1970,6 @@ def get_assessment
     end
   end
 
-  def extend_config_module
```

Comment:
It was already in autograde.rb though.  And assessments_controller.rb.  And submissions_controller.rb.  And it was the same code each time.  Helpers are really meant for views, not general use.  Also, didn't we deprecate the scoreboard headers thing?